app/src/main/java/com/example/parkingsystemjava/mvp/view/ActivityView.java

```diff
@@ -0,0 +1,34 @@
+package com.example.parkingsystemjava.mvp.view;
```

Comment:
this class should be into a package "base" inside the view package

Reply:
done!